website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Column level lineage"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+dbt Explorer now offers column level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a dbt resource (model, source, or snapshot), Explorer provides the full lineage for the data in that column.
```

Comment:
Instead of this:
> For each column in a dbt resource (model, source, or snapshot), Explorer provides the full lineage for the data in that column.

This: 
For each column in a resource (model, source, or snapshot) in a dbt project, Explorer provides the end-to-end lineage for the data in that column given how it's used.